core_lib/src/managers/layermanager.cpp

```diff
@@ -100,6 +100,11 @@ Layer* LayerManager::findLayerByName(QString sName, Layer::LAYER_TYPE type)
     return object()->findLayerByName(sName, type);
 }
 
+Layer* LayerManager::findLayerById(int layerId)
+{
+    return object()->findLayerById(layerId);
```

Comment:
The motivation for me has been to hide the object class as much as possible, so we use the layer manager instead to access layers. As you say, there's no improvement in terms of getting to the function 😅

Reply:
It's encouraged to do so to avoid touching the `object` directly as managers might need to notify UI or other components of the changes. In this specific case, yes there is no difference.